apps/next-evans/pages/friend/[friend_slug].tsx

```diff
@@ -44,11 +38,7 @@ export const getStaticProps: GetStaticProps<Props> = async (context) => {
       },
     },
   });
-  if (!friend) {
-    return {
-      notFound: true,
-    };
-  }
+  invariant(friend !== null);
```

Comment:
using it here too.  as we go forward, i want you to use this to simplify guards/error cases, especially with querying for data at build time 👍